### PR TITLE
feat: add format to Telegram

### PR DESCRIPTION
**Pull request by tanoggy**
Add format support for Telegram (Markdown and HTML format)
Fix some spell
image

### CODE REVIEW

**Comment by tanoggy** [comment]
hi @AbdulTheActivePiecer, please review my pull request

**Comment by abuaboud** [comment]
@allcontributors please add @tanoggy for plugin

File: packages/pieces/telegram-bot/src/lib/action/send-text-message.action.ts [hunk 1]
```diff
@@ -28,23 +31,47 @@ export const telegramSendMessageAction = createAction({
...
+            value: format,
+        }),
         message: Property.LongText({
             displayName: 'Message',
             description: 'The message to be sent',
             required: true,
         })
     },
     async run(ctx) {
+        console.log(ctx)
```
Comment (abuaboud) [inline packages/pieces/telegram-bot/src/lib/action/send-text-message.action.ts]: Please remove this

File: packages/pieces/telegram-bot/src/lib/action/send-text-message.action.ts [hunk 2]
```diff
@@ -28,23 +31,47 @@ export const telegramSendMessageAction = createAction({
...
         message_thread_id: Property.ShortText({
             displayName: 'Message Thread Id',
-            description: 'Unique identifer for the target message thread of the forums; for forums supergroups only',
+            description: 'Unique identifier for the target message thread of the forums; for forums supergroups only',
             required: false,
         }),
+        format: Property.StaticDropdown({
+            displayName: 'Format',
+            description: 'Choose format you want ',
+            required: true,
```
Comment (abuaboud) [inline packages/pieces/telegram-bot/src/lib/action/send-text-message.action.ts]: required: false

File: packages/pieces/telegram-bot/src/lib/action/send-text-message.action.ts [hunk 3]
```diff
@@ -28,23 +31,47 @@ export const telegramSendMessageAction = createAction({
...
     async run(ctx) {
+        console.log(ctx)
         return await httpClient.sendRequest<never>({
             method: HttpMethod.POST,
             url: telegramCommons.getApiUrl(ctx.auth, 'sendMessage'),
             body: {
                 chat_id: ctx.propsValue['chat_id'],
                 text: ctx.propsValue['message'],
                 message_thread_id: ctx.propsValue['message_thread_id'] ?? undefined,
+                parse_mode: ctx.propsValue['format']
```
Comment (abuaboud) [inline packages/pieces/telegram-bot/src/lib/action/send-text-message.action.ts]: Please add `ctx.propsValue['format'] ?? 'MarkdownV2' ` as the field is no longer required

**Review by abuaboud — CHANGES_REQUESTED** [review]
Please also bump the version in package.json to 0.3.4

**Comment by tanoggy** [comment]
> Please also bump the version in package.json to 0.3.4

please review again

**Review by abuaboud — APPROVED** [review]

**Pull request merged**